extensions/wikia/WallNotifications/styles/WallNotifications.globalNavigation.scss

```diff
@@ -6,17 +6,22 @@
 @import 'extensions/wikia/Venus/styles/mixins';
 @import 'extensions/wikia/Venus/styles/typographyMixins';
 
-$header-bg-color: #092140;
+$black-half-transparent: rgba(0, 0, 0, .6);
+$black-transparent: rgba(0, 0, 0, 0);
 $border-color: #c7d1d8;
-$wiki-header-color: #2b54b5;
-$wiki-header-bg-color: rgba(113, 133, 148, .2);
+$header-bg-color: #092140;
+$markasread-bg-color: rgb(9, 35, 68);
+$markasread-color: #fff;
+$notification-bubble-color: rgb(196, 206, 217);
```

Comment:
you can make this color the same as $notifications-divider-color. I didn't have this defined on spec so I tried to reuse the one from search and I made the typo (197 => 196)